### PR TITLE
[FIX][9.0] changing the default SRID (correction of the previous one)

### PR DESCRIPTION
**Pull request by thomaspaulb**
Backport of part of https://github.com/OCA/geospatial/pull/155/commits/f5dfefb7d74a40c1c2684a019c7f1f609b6d8c26

Fixes https://github.com/OCA/geospatial/issues/363

### CODE REVIEW

File: base_geoengine/fields.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class GeoField(Field):
 
     _slots = {
         'dim': 2,
-        'srid': 900913,
+        'srid': 3857,
```
Comment (gfcapalbo) [inline base_geoengine/fields.py:45]: T```
ypeError: Reprojection of column is not implementedWe can not change srid 3857 to 900913
2024-03-05 16:23:39,250 34480 CRITICAL spinpompen-nightlydatabase openerp.service.server: Failed to initialize database `spinpompen-nightlydatabase`.
Traceback (most recent call last):
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/service/server.py", line 894, in preload_registries
    registry = RegistryManager.new(dbname, update_module=update_module)
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/modules/registry.py", line 386, in new
    openerp.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/modules/loading.py", line 376, in load_modules
    force, status, report, loaded_modules, update_module, models_to_check)
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/modules/loading.py", line 271, in load_marked_modules
    perform_checks=perform_checks, models_to_check=models_to_check
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/modules/loading.py", line 155, in load_module_graph
    init_module_models(cr, package.name, models)
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/modules/module.py", line 300, in init_module_models
    result = obj._auto_init(cr, {'module': module_name})
  File "/home/ubuntu/odoo/custom/src/odoo/openerp/api.py", line 250, in wrapper
    return old_api(self, *args, **kwargs)
  File "/home/ubuntu/odoo/auto/addons/base_geoengine/geo_model.py", line 55, in _auto_init
    fct(cursor, kol, geo_columns[kol], self._table, self._name)
  File "/home/ubuntu/odoo/auto/addons/base_geoengine/geo_field.py", line 150, in update_geo_column
    geo_column._srid, check_data[0]))
TypeError: Reprojection of column is not implementedWe can not change srid 3857 to 900913




```
will add dropping column in pre-commit-

File: geoengine_base_geolocalize/migrations/9.0.0.2.0/post-migrate.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2024 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import SUPERUSER_ID, api
```
Comment (gfcapalbo) [inline geoengine_base_geolocalize/migrations/9.0.0.2.0/post-migrate.py]: File "/home/ubuntu/odoo/auto/addons/geoengine_base_geolocalize/migrations/9.0.0.2.0/post-migrate.py", line 5, in <module>
    from odoo import SUPERUSER_ID, api
ImportError: cannot import name SUPERUSER_ID

**Review by gfcapalbo — COMMENTED** [review]
@thomaspaulb 
Found a couple of errors on install , fixed.
Deployed on test and nightly, but still some small errors on update when changing srid happen.

will fix tomorrow morning

**Comment by thomaspaulb** [comment]
@gfcapalbo Maybe another review

**Comment by thomaspaulb** [comment]
@gfcapalbo @pedrobaeza Ready for another review

File: base_geoengine/__openerp__.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # © 2016 Yannick Vaucher (Camptocamp SA)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {'name': 'Geospatial support for Odoo',
- 'version': '9.0.1.2.5',
+ 'version': '9.0.1.3.6',
```
Comment (pedrobaeza) [inline base_geoengine/__openerp__.py]: Why this random version? Just bump the next one, not one minor + one revision more

**Pull request closed without merging**